compiler/rustc_ast/src/ast.rs

```diff
@@ -2562,6 +2566,25 @@ impl BoundConstness {
     }
 }
 
+/// The asyncness of a trait bound.
+#[derive(Copy, Clone, PartialEq, Eq, Encodable, Decodable, Debug)]
+#[derive(HashStable_Generic)]
+pub enum BoundAsyncness {
+    // `Type: Trait`
```

Comment:
```suggestion
    /// `Type: Trait`
```

;)

Reply:
I don't think I'll ever successfully leave a doc comment without accidentally making it a normal comment first.